vc_assistance/indico_vc_assistance/client/style.scss

```diff
@@ -0,0 +1,22 @@
+// This file is part of the CERN Indico plugins.
+// Copyright (C) 2014 - 2018 CERN
+//
+// The CERN Indico plugins are free software; you can redistribute
+// them and/or modify them under the terms of the MIT License; see
+// the LICENSE file for more details.
+@import 'base/palette';
+
+.overview-results {
+
+    td.request-state {
+        width: 1%;
+
+        .icon-checkmark {
```

Comment:
I don't think it makes sense to show the checkmark/state since this request type is auto-accepted.

BTW, should we consider adding something in the core to mark a request as "no approval required"? This is now the second plugin that doesn't require approval and there are a few things like showing "Accepted" on the request card which don't make much sense for a request with no approval workflow.

Reply:
No, we didn't consider it. What do you think @pferreir?

Reply:
Agreed. That would make things easier to understand. Let's open a task?

Reply:
https://github.com/indico/indico/issues/3677